lambdas/service/app.py

```diff
@@ -780,7 +780,7 @@ def file_url(uuid, **params):
 
 def self_url(endpoint_path=None):
     protocol = app.current_request.headers.get('x-forwarded-proto', 'http')
-    base_url = app.current_request.headers['host']
+    base_url = app.current_request.headers.get('host')
```

Comment:
We didn't need this until now. Furthermore, if the `host` header were missing, I would want to learn that here rather than …